pytravharv/WebAccess.py

```diff
@@ -43,7 +44,7 @@ def handle_data(self, data):
             self.scripts.append({self.type: data})
 
 
-def WebAccess(url):
+def fetch(url):
     triplestore = Graph()
     download_uri_to_store(url, triplestore)
```

Comment:
why not make a rephrase of the previous method_name to include `_into_graph`

they seem functionally totally equivalent, except the creation vs the reuse of the graph to be used?

```
def _find_description_into_graph(subject: str, graph: Graph) -> None:
```